service-mirror.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	apiError "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+func main() {
+	// Set up logger .
+	log := &logrus.Logger{
+		Out:   os.Stderr,
+		Level: logrus.DebugLevel,
+		Formatter: &logrus.TextFormatter{
+			TimestampFormat: "2006-01-02 15:04:05",
+			ForceColors:     true,
+			DisableColors:   false,
+			FullTimestamp:   true,
+		},
+	}
+	//Shows line number: Too long
+	// log.SetReportCaller(true)
+
+	log.Info("-- Starting Global Mirror ---")
```

Comment:
this might look a bit odd since afaik logrus uses structured logging. I'd perhaps remove the `---`

Reply:
Ok ok wanted some sort of banner but make sense.